buildSrc/createHtml.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Utility to create the HTML landing page for the app.
+ */
+import fs from "fs-extra"
+import {renderHtml} from "./LaunchHtml.js"
+import {mkdir} from "fs/promises"
+import path from "path"
+
+/**
+ *
+ * @param env Object with the following keys:
+ * 	mode: String of value "App" or "Desktop" or "Browser"
+ * 	dist: Boolean
+ * 	staticUrl: String defining app base url for non-production environments
+ * 	versionNumber: String containing the app's version number
+ * @returns {Promise<Awaited<void>[]>}
+ */
+export async function createHtml(env) {
```

Comment:
changes here?

Reply:
Nope, nothing worth mentioning anyway. env.js(), createHtml() and buildWebapp() have hardly changed. Those three files should be looked at and cleaned up as a next step - I have added some comments to the original issue.